packages/matter.js/src/device/PairedNode.ts

```diff
@@ -374,7 +374,12 @@ export class PairedNode {
             updateTimeoutHandler: async () => {
                 logger.info(`Node ${this.nodeId}: Subscription update not received ...`);
                 try {
-                    await this.subscribeAllAttributesAndEvents();
+                    const { attributeChangedCallback, eventTriggeredCallback } = this.options;
+                    await this.subscribeAllAttributesAndEvents({
+                        ignoreInitialTriggers: true,
+                        attributeChangedCallback: data => attributeChangedCallback?.(this.nodeId, data),
```

Comment:
Why you change here and next line the callback method? In fact what's missing is thatched original options are also set when resubscribing. So passing the callbacks from options just like in options of the subscribeAllAttributesAndEvents method should be sufficient or?

Reply:
Yeah, my oversight.